helm-go-lib/main.go

```diff
@@ -277,3 +275,59 @@ func (f *CustomConfigFlags) ToRESTConfig() (*rest.Config, error) {
 
 	return c, nil
 }
+
+//export pullChart
```

Comment:
Is this `//export` the load-bearing comment you had mentioned in standup?  Should this have _another_ comment explaining that it's loadbearing?

Reply:
I'm updating the README for that.

Reply:
It's load bearing as we found out.. I was trying to understand if it's a JNA thing, a Go thing, or both 🥲

Reply:
Is there any way to get some kind of automated guardrail here (a test or something?) that will start failing if the loadbearing comment gets removed?  It just looks like the kind of thing that inevitably gets cleaned out by accident, README or no.

Reply:
Seems we're only exporting `updateAndPullChart` to the Scala code -- should we remove this comment?